micro_dl/config_inference_dataset.yml

```diff
@@ -0,0 +1,69 @@
+dataset:
+  augmentation: false
+  data_dir: /hpc/projects/comp_micro/projects/virtualstaining/torch_microDL/data/2022_09_07_Soorya_KidneyRickettsia/data/
+  input_channels:
+  - 3
+  mask_channels:
+  - 4
+  model_task: regression
+  normalize: false
+  random_seed: 42
+  split_by_column: pos_idx
+  split_ratio:
+    test: 0.05
+    train: 0.05
+    val: 0.9
+  target_channels:
+  - 0
+  train_fraction: 1
+network:
+  activation:
+    type: relu
+  batch_norm: true
+  block_sequence: conv-activation-bn
+  class: UNetStackTo2D
+  data_format: channels_first
+  depth: 5
+  dropout: 0.2
+  filter_size: 3
+  final_activation: linear
+  height: 2048
```

Comment:
maybe move width to be near height?

Reply:
Yeah... These config files have some wonky stuff in them left over from the master branch. Soorya tackled a lot of this in the documentation pr, so I will pull those changes.